src/app/wallets/get-on-chain-fee.ts

```diff
@@ -38,49 +52,22 @@ export const getOnChainFee = async ({
   if (balance instanceof Error) return balance
 
   // avoids lnd balance sniffing attack
-  if (balance < amount) return new InsufficientBalanceError("Balance is too low")
+  if (balance < amountChecked) return new InsufficientBalanceError("Balance is too low")
 
   const onChainService = OnChainService(TxDecoder(BTC_NETWORK))
   if (onChainService instanceof Error) return onChainService
 
-  const onChainFee = await onChainService.getOnChainFeeEstimate({
-    amount,
+  const minerFee = await onChainService.getOnChainFeeEstimate({
```

Comment:
maybe `networkFee` should be a more generic name (it works for lightning too)

Reply:
that's a great idea

Reply:
I'm not too sure this should be networkFee at the end, at least at this level. The reason being that routingFee and minerFee are different. One we always know how much we will pay in advance (minerFee), the other this is not always known (always know if we probe first succesfully)